cypress/integration/ete/reset_password.spec.js

```diff
@@ -2,34 +2,44 @@
 
 describe('Password reset flow', () => {
   context('Account exists', () => {
-    // This test depends on this Mailslurp account already being registered
+    // This test depends on this Mailosaur account already being registered
     const existing = {
-      email: 'a1ae9048-2823-45a0-a0e1-89e8f7a22420@mailslurp.com',
-      inbox: 'a1ae9048-2823-45a0-a0e1-89e8f7a22420',
+      serverId: 'ytcp0n8q',
+      serverDomain: 'ytcp0n8q.mailosaur.net',
```

Comment:
do we want to put the server id in the environment configuration?

As this is publicly available, so it's possible that someone could spam the inbox?

Reply:
good precaution, I'll change to do this.

 If you get a moment later could you how we can update the Dev .env files in aws to include it?